user/caching.md

```diff
@@ -76,6 +76,32 @@ In the cases where this is not desirable, you can use specify the [arbitrary dir
 to get around it.
 See [this GitHub issue](https://github.com/travis-ci/travis-ci/issues/2518) for more information.
 
+### cache RVM Ruby version for non Ruby projects
+
+There are projects using machines not based on Ruby but having some Ruby executions. For example, a NodeJS application that has a Ruby functional test suite:
+
+```
+sudo: required
+dist: trusty
+
+language: node_js
+
+node_js:
+  - "6"
+
+script:
+  - npm test
+  - cucumber features/
+```
+
+For these cases installing a version of ruby with `rvm install 2.3.1` may take more than 3 minutes. For these cases you can cache the ruby installation.
+
+```yaml
+ cache:
+    directories:
+     - /home/travis/.rvm/
+```
```

Comment:
@plaindocs do you mean removing lines from 99 to 103?

I am okay with it. But this are the lines that people will have to add to their YML to make it work, I think they are valuable. Actually, I feel they give more value than lines from 83 to 95. 

Still, I am not documentation expert. So feel free to modify it as you think is the best for TravisCI

Reply:
I'm talkng about 83-95 - thanks